app/Http/Controllers/UserPreferencesController.php

```diff
@@ -118,4 +129,13 @@ private function getValidLocales()
             }
         }, []);
     }
+
+    private function getValidStyles()
+    {
+        $site_styles = Config::getDefinitions()['site_style']['options'];
```

Comment:
Best way is:
```php
$definitions = new DynamicConfig();
return $defintions->get('site_style')->getOptions();
```

Reply:
Why are you sorting them? I'd suggest re-ordering them in the definition instead.

Reply:
changed :-)